atlascope/core/rest/endpoints/investigation_endpoints.py

```diff
@@ -89,3 +90,9 @@ def permissions(self, request, pk=None):
         payload = InvestigationDetailSerializer(investigation).data
         payload = {k: v for k, v in payload.items() if k in ['owner', 'investigators', 'observers']}
         return Response(payload, status=status.HTTP_200_OK)
+
+    @object_permission_required(edit_access=True)
+    @action(detail=True, methods=['GET'])
+    def pins(self, request, pk=None):
```

Comment:
Could you add a `@swagger_auto_schema` wrapper here to clarify the response schema? Right now it just shows up as the `InvestigationDetail` on the swagger page.

Reply:
Changed in 722402a